code/code/cmd/cmd_message.cc

```diff
@@ -538,7 +538,7 @@ void TMessages::initialize()
         tChar;
   FILE *tFile = NULL;
 
-  sprintf(tString, "player/%c/%s.strings",
+  sprintf(tString, "../mutable/player/%c/%s.strings",
```

Comment:
I assume the migration path is:

- Stop Sneezy
- Copy /home/sneezy/sneezymud-docker/sneezymud/lib /home/sneezy/sneezymud-docker/mutable/
- Start Sneezy 


Would it be better to make the location of this directory a configuration parameter in sneezy.cfg? data_directory or something?

Reply:
Yeah, that should work fine for migration — for new setup, you wouldn't even need to copy. Sneezy ought to create required dirs automatically.

Nope — it'd be better to migrate all this into DB.

Reply:
Yeah I assumed this was a "get it working so we can put off migrating it to the DB at least one more day" kind of effort.

I'm fine with the work you've done here as long as I can get my test container to work without losing all my pfiles.